api/test/unit/controllers/test_search_controller.py

```diff
@@ -13,36 +13,51 @@
 
 
 @pytest.mark.parametrize(
-    "total_hits, real_result_count, page_size, expected",
+    "total_hits, real_result_count, page_size, page, expected",
     [
         # No results
-        (0, 0, 10, (0, 0)),
+        (0, 0, 10, 0, (0, 0)),
         # Setting page size to 0 raises an exception
         pytest.param(
-            0, 0, 0, (0, 0), marks=pytest.mark.raises(exception=ZeroDivisionError)
+            0, 0, 0, 0, (0, 0), marks=pytest.mark.raises(exception=ZeroDivisionError)
         ),
         # Fewer results than page size leads to max of result total
-        (5, 5, 10, (5, 0)),
+        (5, 5, 10, 0, (5, 0)),
```

Comment:
I don't understand why the page_count returned here is 0. Isn't it supposed to be 1 even if we have 1 result?

Reply:
It should be, yes. I suppose it should only ever return 0 if there are 0 results. I'll look into this, thanks for catching it.

Reply:
This turned out to mostly be a test quirk, as the page sent to the method was `0` even though that's not a real page that ever really gets requested because we do not 0 index the pages. However, it's easy to correct for it by treating `page = 0` the same as `page = 1`.

Reply:
Thank you for looking into this, @sarayourfriend! Do you think a comment about it would be useful in the tests?